tests/testdata/programs/auto-mode-upgrade/README.md

```diff
@@ -0,0 +1,3 @@
+# EKS Auto Mode
+
+TODO
```

Comment:
Sketch a few sentences here?

Reply:
Whoops, this should actually be removed. Was a left over from copying the test setup. Good catch!